templates/spark/v1/spark.yaml

```diff
@@ -0,0 +1,6 @@
+imports:
```

Comment:
Althought then I think you get into even more annoyance of not having a local registry, because you end up launching this file as `example.yaml`, and have one of those things running around, which is weird.